circle.yml

```diff
@@ -14,10 +14,7 @@ machine:
 dependencies:
   override:
     - pip -V
-    - pip install -U pip
-    - pip install -U ipdb
-    - pip install -U tox
-    - pip install -U .
+    - pip install -U 'pip<8.0' 'virtualenv<14.0' ipdb tox .
```

Comment:
this ensures that the py32 virtualenv is created with the correct version of pip